build.sbt

```diff
@@ -22,22 +21,13 @@ val compilerOptions = Seq(
 
 scalacOptions in ThisBuild ++= compilerOptions
 
-val minJacksonVersion: String = "2.10.1"
-val minJacksonLibs = Seq(
-  "com.fasterxml.jackson.core" % "jackson-core" % minJacksonVersion,
-  "com.fasterxml.jackson.core" % "jackson-annotations" % minJacksonVersion,
-  "com.fasterxml.jackson.dataformat" % "jackson-dataformat-cbor" % minJacksonVersion,
-  "com.fasterxml.jackson.core" % "jackson-databind" % minJacksonVersion,
-  "com.fasterxml.jackson.datatype" % "jackson-datatype-jsr310" % minJacksonVersion
-)
```

Comment:
I assume this was added to force a more recent version of Jackson by making it a direct dependency? This no longer needed because a more recent version of Jackson is now being requested as a transitive dependency?

Reply:
>I assume this was originally added to force a more recent version of Jackson by making it a direct dependency?

Yes

>This is now no longer needed because a more recent version of Jackson is now being requested as a transitive dependency?

It should! I haven't checked. But next  up is Play 2.8, so I recommend we look again then.